tests/conftest.py

```diff
@@ -2592,3 +2592,20 @@ def ns_resources_and_connections_cleanup():
     request.addfinalizer(ns_resources_and_connections_cleanup)
 
     return _create_ns_resources
+
+
+@pytest.fixture(scope="session", autouse=True)
+def collect_logs_fixture(request):
+    """
+    This fixture collects ocs logs after tier execution and this will allow
+    to see the cluster's status after the execution on all execution status options.
+    """
+    def finalizer():
+        """
+        Tracking both logs separately reduce changes of collision
+        """
+        if not config.RUN['cli_params'].get('deploy') and not config.RUN['cli_params'].get('destroy'):
```

Comment:
destroy -> teardown as we don't have any destroy parameter but --teradown https://ocs-ci.readthedocs.io/en/latest/docs/usage.html that's why we see the must gather collected also for destroy job.
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-destroy-ocs-cluster/9714/console

Reply:
Fix of that verified here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/12626/
(Only one folder generated)